src/seedu/addressbook/parser/Parser.java

```diff
@@ -11,17 +11,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import seedu.addressbook.commands.AddCommand;
-import seedu.addressbook.commands.ClearCommand;
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```

Comment:
According to the style guide, you should use wildcard import even though it is quite verbose.

Reply:
I don't quite agree with this, could you please clarify further? 
Won't it contradict the idea of refactoring? (I.E. not reducing the amount of lines and making it readable)

Reply:
Sorry, there was a typo there. You should **not** do wildcard import. 

Ref: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

Under no circumstances, you should do wildcard import.

Reply:
Noted, weird convention but thanks!